components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/constants/SQLConstants.java

```diff
@@ -1012,6 +1012,16 @@
             "   AND APP.ORGANIZATION = ? " +
             "   AND APP.SUBSCRIBER_ID = SUB.SUBSCRIBER_ID";
 
+    public static final String GET_APPLICATION_ID_PREFIX_FOR_GROUP_COMPARISON =
```

Comment:
Don't unnecessarily split a string and join with `+` as it recreates new String objects for each concatenation. If necessary use `StringBuilder` instead.